lang/test/org/partiql/lang/syntax/SqlParserTest.kt

```diff
@@ -4268,4 +4268,26 @@ class SqlParserTest : SqlParserTestBase() {
             System.currentTimeMillis() - startTime < maxParseTime
         )
     }
+
+    @Test
+    fun testOrderByMetas() {
+        // Arrange
+        val query = "SELECT * FROM << { 'x': 2 } >> ORDER BY x"
+        val expected = SourceLocationMeta(1, 32, 5)
+
+        // Act
+        val stmt = parser.parseAstStatement(query)
+
+        // Gather Metas and Assert
+        val expr = when (stmt) {
+            is PartiqlAst.Statement.Query -> stmt.expr
+            else -> throw AssertionError("Expected a PartiqlAst.Statement.Query")
+        }
+        val orderExpr = when (expr) {
+            is PartiqlAst.Expr.Select -> expr.order
+            else -> throw AssertionError("Expected query with an ORDER BY clause")
+        }
+        val metas = orderExpr?.metas ?: throw AssertionError()
```

Comment:
This last check for metas could have a better error message. Consider:

```suggestion
        val metas = orderExpr?.metas ?: throw AssertionError("Expected ORDER BY clause to have metas")
```